tornado/http1connection.py

```diff
@@ -131,7 +131,7 @@ def __init__(
         self.no_keep_alive = params.no_keep_alive
         # The body limits can be altered by the delegate, so save them
         # here instead of just referencing self.params later.
-        self._max_body_size = self.params.max_body_size or self.stream.max_buffer_size
+        self._max_body_size = self.params.max_body_size if self.params.max_body_size != None else self.stream.max_buffer_size
```

Comment:
I can't imagine there's much of a performance difference but it's usually more idiomatic to use `is not None` instead of `!= None` since `None` is a singleton. Another reason to use `is not None` is that `==`/`!=` behavior can be overwritten. Example:

```pycon
In [1]: class Foo:
   ...:     def __eq__(self, other):
   ...:         if other is None:
   ...:             return True
   ...:         return False
   ...:

In [2]: foo = Foo()

In [3]: foo == None
Out[3]: True

In [4]: foo is None
Out[4]: False
```

Reply:
Looks like this is in fact one of the linter errors that @bdarnell is referring to.